_sections/30-bindings.md

```diff
@@ -41,6 +41,7 @@ The following attributes are supported on `<bind>` nodes. Only the nodeset attri
 | `constraint`| Specifies acceptable answers for the specified prompt with an XPath expression. Will only be evaluated when the node is non-empty.
 | `calculate` | Calculates a node value with an XPath expression.
 | `saveIncomplete` | Specifies whether to automatically save the draft record when the user reaches this question, options `true()` and `false()`. Considered false() if omitted.
+| `jr:requiredMsg` | Specifies the custom message to be displayed when the `required` is violated. Attribute in "http://openrosa.org/javarosa" namespace.
```

Comment:
For the namespace, is the goal to specify it at the beginning of lists that are in a certain namespace (here, `constraintMsg`, `preload`, etc are in the same namespace)? I'm wondering whether it might be good enough to mention once at the top.